packages/caliper-fabric/test/sample-configs/NetworkConfigNotMutual.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Looks like this file has not been updated as the suggested changes are not present